std_msgs/Header header
FixType fix_type